DataManagementSystem/Client/DataManager.py

```diff
@@ -1506,19 +1506,39 @@ def put( self, lfn, fileName, diracSE, path = None ):
   # def putReplica(self,lfn,storageElementName,singleFile=False):
   # def replicateReplica(self,lfn,size,storageElementName,singleFile=False):
 
-  def getActiveReplicas( self, lfns, getUrl = True ):
+  def getActiveReplicas( self, lfns, getUrl = True, diskOnly = False, preferDisk = False ):
```

Comment:
I don't really se the interest of preferDisk here ? it does not really seem used to me ?